devel/tests/test_user.py

```diff
@@ -1,8 +1,8 @@
 from django.contrib.auth.models import User
 from django.test import TestCase
 
-from .utils import UserFinder
-from .models import UserProfile
+from devel.utils import UserFinder
+from devel.models import UserProfile
```

Comment:
I appreciate the renaming here, but why?
Needs to be covered in https://github.com/archlinux/archweb/pull/42.